airbyte-webapp/src/components/connectorBuilder/StreamTestingPanel/SchemaDiffView.module.scss

```
@@ -0,0 +1,20 @@
+@use "scss/variables";
+@use "scss/colors";
+
+.diffLine {
+  margin: 0;
+  padding: 0 variables.$spacing-sm;
+}
+
+.added {
+  background-color: colors.$green-100;
+}
+
+.removed {
+  background-color: colors.$red-100;
+}
+
+.infoBox {
+  align-items: flex-start;
+  width: 100%;
+}
```

```suggestion
.infoBox {
  align-items: flex-start;
  width: 100%;

  svg {
    color: colors.$yellow;
  }
}
```
Just a small suggestion to color the warning icon here more similarly to the other schema warning icons. Can either take it or leave it, up to you